docs/samples/pvc/tensorflow.yaml

```diff
@@ -0,0 +1,11 @@
+apiVersion: "serving.kubeflow.org/v1alpha1"
+kind: "KFService"
+metadata:
+  name: "flowers-sample"
+spec:
+  default:
+    tensorflow:
+      modelUri: "/mnt/models/tensorflow/flowers"
+      persistentVolumeClaim:
```

Comment:
What if we baked this into the model uri? 

s3://...
gcs://...
pvc://name/path

Reply:
Great idea, but if so here has a problem, we need to put 3 parts in the model uri: PVC name, PVC mount path, model path. So we need to separate 3 parts using a symbol and the symbol cannot be included in the model uri and OS path. Checked the PVC name rules and OS path rules, we may use ";" to seprate that, for example:

`pvc://pvc-name;/pvc/mount/path;/pvc/mount/path/mode/path`

Comments? or better suggestion? Thanks a lot!

Reply:
Hi @ellis-bigelow Do you think is that better way to use `annotations`? if user want to loading model from pvc, user need to add two `annotations` such as `pvcName: kfserving-pvc` and `pvcMountPath: /mnt`, and the `modelUri` will be `/mnt/model/export/path`. And no need to introuduce new types. Thanks.